execution_engine/src/storage/global_state/in_memory.rs

```diff
@@ -28,18 +30,20 @@ use crate::storage::{
     },
 };
 
+/// Global state implemented purely in memory only. No state is saved to disk, and this is mostly
+/// used for testing purposes.
 pub struct InMemoryGlobalState {
-    pub environment: Arc<InMemoryEnvironment>,
-    pub trie_store: Arc<InMemoryTrieStore>,
-    pub protocol_data_store: Arc<InMemoryProtocolDataStore>,
-    pub empty_root_hash: Blake2bHash,
+    pub(crate) environment: Arc<InMemoryEnvironment>,
```

Comment:
```suggestion
    /// Environment for `InMemoryGlobalState`.
    /// Basically empty because this global state does not support transactions.
    pub(crate) environment: Arc<InMemoryEnvironment>,
```

Reply:
Chatting with @dwerner , even though this is private we should document this for others who may review it.